docs/pages/play.vue

```diff
@@ -257,6 +279,73 @@ export default {
       // Check if editors contain default JS and Template
       return this.js.trim() === defaultJS.trim() && this.html.trim() === defaultHTML.trim()
     },
+    codepen_data() {
+      const data = {
+        editors: '101',
+        layout: 'left', // left, right, top
+        html_pre_processor: 'none',
+        css_pre_processor: 'none',
+        css_prefix: 'autoprefixer',
+        js_pre_processor: 'babel',
+        head: '<meta name="viewport" content="width=device-width">',
+        css_external: [
+          '//unpkg.com/bootstrap/dist/css/bootstrap.min.css',
+          '//unpkg.com/bootstrap-vue@latest/dist/bootstrap-vue.css'
+        ].join(';'),
+        js_external: [
+          '//unpkg.com/babel-polyfill@latest/dist/polyfill.min.js',
+          '//unpkg.com/vue@latest/dist/vue.min.js',
+          '//unpkg.com/bootstrap-vue@latest/dist/bootstrap-vue.js'
+        ].join(';'),
+        html: this.fiddle_html,
+        js: this.fiddle_js,
+        css: 'body { padding: 1rem; }'
+      }
+      return JSON.stringify(data)
+    },
+    codesandbox_data() {
+      const html = this.html.trim() || '<div></div>'
+      const js = this.js.trim() || '{}'
+      const vueContent = dedent`
+        <template>
+          ${html}
+        </template>
+
+        <script>
+          export default ${js}
+        <\/script>
+      `.replace('\\/', '/')
+      const htmlContent = '<div id="app"></div>'
+      const jsContent = dedent`
+        import Vue from 'vue'
+        import BootstrapVue from 'bootstrap-vue'
+        import App from './App'
+
+        import 'bootstrap/dist/css/bootstrap.css'
+        import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+        Vue.use(BootstrapVue)
+
+        new Vue({
+          el: '#app',
+          render: h => h(App)
+        })
+      `
+      const dependencies = {
+        bootstrap: 'latest',
+        'bootstrap-vue': 'latest',
```

Comment:
And was thinking if these exports (jsFiddle, CodePen, CodeSandbox) should use the version number in the docs rather than `latest`?

Reply:
Yeah, it should be possible to use the `parseVersion()` util here again for that.

Reply:
Done.